app/src/app/components/stix/stix-list/stix-list.component.ts

```diff
@@ -437,9 +450,18 @@ export class StixListComponent implements OnInit, AfterViewInit, OnDestroy {
             let target_ref = element['target_ref'];
             // Get type to navigate from target_ref
             let type = this.typeMap[target_ref.split('--')[0]];
-
             this.router.navigateByUrl('/'+ type + '/' + target_ref);
         }
+        else if (this.config.clickBehavior && this.config.clickBehavior == "linkToObjectRef") {
+          // technically a note can be linked to many objects, we will select the first object
+          let object_ref = element['object_refs'][0];
```

Comment:
Good catch! A note can _technically_ be associated with many objects (according to the STIX specification), but our use of it is only one-to-one, so there will only be a single object in the list.